pkg/velaql/providers/query/handler.go

```diff
@@ -54,6 +54,14 @@ const (
 	ProviderName = "query"
 	// HelmReleaseKind is the kind of HelmRelease
 	HelmReleaseKind = "HelmRelease"
+	// HTTPS https protocol name
+	HTTPS = "https"
+	// HTTP http protocol name
+	HTTP = "http"
+	// Mysql mysql protocol name
+	Mysql = "mysql"
+	// Redis redis protocol name
+	Redis = "redis"
 )
```

Comment:
These constants might not be directly associated with velaql. Suggest moving them out somewhere.

Reply:
the ServiceEndpoint struct is defined here. I have found no suitable package, any suggestions?

Reply:
Maybe move to another new file? Like pkg/utils/constants.go?